app/assets/stylesheets/custom.css.scss

```diff
@@ -151,3 +157,34 @@ aside {
   margin-top: 15px;
 }
 
+/* forms */
+
+input, textarea, select, .uneditable-input {
+  border: 1px solid #bbb;
+  width: 100%;
```

Comment:
Properties should be ordered border, margin-bottom, width<br>Each selector in a comma sequence should be on its own line